vajram/src/main/java/com/flipkart/krystal/vajram/inputs/ResolveDep.java

```diff
@@ -4,8 +4,8 @@
 import java.lang.annotation.RetentionPolicy;
 
 @Retention(RetentionPolicy.RUNTIME)
-public @interface Resolve {
-  String value();
+public @interface ResolveDep {
```

Comment:
Now that `value` -> `depName` and `inputs` -> `depInputs` both have `dep` in them, `Resolve` doesn't need a `Dep`: it seems redundant.

Example:

```java
import com.flipkart.krystal.vajram.ComputeVajram;
import com.flipkart.krystal.vajram.inputs.Resolve;

abstract class X implements ComputeVajram<String> {

  @Resolve(depName="productData", depInputs = "fsn")
  static String computeFsnForProductData(@Using("productId") String productId){
    return productId;
  }
} 
```

Reply:
Changed it to Resolve